From b48c0018d1de61a77e59355eef6d8ed7d6e5aa9d Mon Sep 17 00:00:00 2001
From: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
Date: Fri, 30 Jul 2021 02:49:33 +0000
Subject: [PATCH] Mailbox: Fix compile error caused by spell mistake of
 spin_lock_irqrestore

Replace spin_lock_irqstore() with spin_lock_irqrestore().

Fixes: 4f52df83cd44("Mailbox: Replace mutex lock with spin lock")
Upstream-Status: pending
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 drivers/mailbox/mvl_mhu.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/mailbox/mvl_mhu.c b/drivers/mailbox/mvl_mhu.c
index 3caf04a2d557..e8639b570130 100644
--- a/drivers/mailbox/mvl_mhu.c
+++ b/drivers/mailbox/mvl_mhu.c
@@ -202,7 +202,7 @@ static bool mvl_mhu_last_tx_done(struct mbox_chan *chan)
 
 	spin_lock_irqsave(&mhu->link_lock, flags);
 	val = readq_relaxed(mhu->base + SCP_TO_AP0_MBOX_RINT);
-	spin_unlock_irqstore(&mhu->link_lock, flags);
+	spin_unlock_irqrestore(&mhu->link_lock, flags);
 
 	dev_dbg(mhu->dev, "%s\n", __func__);
 
@@ -216,7 +216,7 @@ static int mvl_mhu_send_data(struct mbox_chan *chan, void *data)
 
 	spin_lock_irqsave(&mhu->link_lock, flags);
 	writeq_relaxed(DONT_CARE_DATA, mhu->base + AP0_TO_SCP_MBOX);
-	spin_unlock_irqstore(&mhu->link_lock, flags);
+	spin_unlock_irqrestore(&mhu->link_lock, flags);
 
 	return 0;
 }
-- 
2.29.2

